Governance.md

```diff
@@ -0,0 +1,78 @@
+# Governance of the rules_scala
+
+These bazel rules for scala are under the bazelbuild org but governed by a
+contributors from the community.
+
+## Goals of this document
+
+Clarify:
+0. values of the project.
+1. how to get a PR merged
+2. how long you should need to wait for a review
+3. how to resolve conflicts in project direction
+4. how to make larger and disruptive changes
+
+## Values of rules_scala
+
+1. Correctness: bazel values fully reproducible builds which are reliable and hermetic (only depend on the current state of your repository)
+2. Speed: we want to build as fast as possible. In a large repository, we believe caching and remote parallel builds are the only way to achieve this. Without correctness, we can't cache and use remote builds, so correctness enables speed. We will almost never trade off correctness for speed.
```

Comment:
How about:
> Speed: we want to build as fast as possible. We also believe caching and remote parallel builds are the only way to achieve this in a large repository. Because caching and work distribution demand correctness, will almost never trade off correctness for speed.

Reply:
I feel like the improvement here is it communicates the same thing more succinctly. It effectively blends the last two sentences together.

Reply:
ok. Like I said I'm not it's an improvement but I can live with the suggestion.